content/applications/inventory_and_mrp/maintenance/maintenance_reporting.rst

```diff
@@ -0,0 +1,295 @@
+:icon:`fa-caret-down` :guilabel:`(caret down)` icon, to the right of the search bar at the top of
+the page.
+
+To group the requests in the report by specific groupings, click the :icon:`oi-group`
+:guilabel:`(Group By)` icon, and select the desired grouping. To add a custom grouping, click
+:guilabel:`Add Custom Group`, and select an option from the resulting drop-down menu.
+
+.. image:: maintenance_reporting/maintenance-reporting-search-bar-menu.png
+   :align: center
+   :alt: Favorites, Group By, and Favorites in search bar drop-down menu.
+
+The left-hand column of the resulting drop-down menu lists all the different :guilabel:`Filters`
+users can select. By default, :guilabel:`Active` is selected, so all open requests are displayed.
+
+.. tip::
+   To add a custom filter to the :guilabel:`Maintenance Requests` report, click :guilabel:`Add
+   Custom Filter`, under the :guilabel:`Filters` section of the drop-down menu. This opens an
+   :guilabel:`Add Custom Filter` pop-up window.
+
+   From this pop-up window, configure the properties of the new rule for the filter. Once ready,
+   click :guilabel:`Add`.
+
+The right-hand column of the drop-down menu lists the :guilabel:`Favorites`, or any searches that
+have been saved as a favorite to be revisited at a later date.
+
+To save a new :guilabel:`Favorite` search, select the desired :guilabel:`Filters`. Then, click
+:guilabel:`Save current search`. In the field directly below :guilabel:`Save current search`, assign
+a name to the search.
+
+Under the assigned name, there are two options, to save the current search either as the
+:guilabel:`Default filter`, or as a :guilabel:`Shared` filter.
+
+Selecting :guilabel:`Default filter` sets this filter as the default when opening this calendar
+view.
+
+Selecting the :guilabel:`Shared` filter makes this filter available to other users.
+
+Once ready, click :guilabel:`Save`. When clicked, the new :guilabel:`Favorite` filter appears in the
+:guilabel:`Favorites` column, and a gold :icon:`fa-star` :guilabel:`(gold star)` icon appears with
+the filter's name in the search bar.
+
+.. _maintenance/reporting/view-types:
+
+Reporting views
+===============
+
+The :guilabel:`Maintenance Requests` report is available in six different views, appearing in order
+from left to right: :guilabel:`Graph` (default), :guilabel:`Pivot`, :guilabel:`Kanban`,
+:guilabel:`List`, :guilabel:`Calendar`, and :guilabel:`Activity`.
+
+.. image:: maintenance_reporting/maintenance-reporting-view-types.png
+   :align: center
+   :alt: Different view type icons for maintenance requests report.
+
+Graph view
+----------
+
+Click the :icon:`fa-area-chart` :guilabel:`(area chart)` icon to change to graph view.
+
+With the graph view selected, the following options appear between the search bar and visual
+representation of the data. These graph-specific options are located to the right of the
+:guilabel:`Measures` and :guilabel:`Insert in Spreadsheet` buttons.
```

Comment:
I think you can remove the 'Measures' reference since it's to the immediate right of the Insert in Spreadsheet?

Reply:
Good call - Suggestion implemented ✅